src/main/resources/migrations.xml

```diff
@@ -73,4 +73,11 @@
         </addColumn>
     </changeSet>
 
+    <changeSet id="add token_type column to tokens table" author="">
+        <addColumn tableName="tokens">
+            <column name="token_type" type="varchar(36)" defaultValue="CREDIT_CARD">
```

Comment:
I would just put CARD since payments are either for credit or debit cards

Reply:
Not having a strong opinion about that - it's true that we have also debit cards, but in our usual lingo we tend to say Credit Cards vs Direct Debit.
Will update